website/views.py

```diff
@@ -114,10 +114,9 @@ def serialize_node_summary(node, auth, primary=True, show_path=False):
             'parent_title': parent_node.title if parent_node else None,
             'parent_is_public': parent_node.is_public if parent_node else False,
             'show_path': show_path,
-            # Read nlogs annotation if possible
-            'nlogs': node.nlogs if hasattr(node, 'nlogs') else node.logs.count(),
             'contributors': contributor_data['contributors'],
             'others_count': contributor_data['others_count'],
+            'description': node.description if len(node.description) < 150 else node.description[0:150] + '...',
```

Comment:
I presume you've verified the edge cases on this, yes?

Reply:
IIRC, yes, but it has been a month, so...